tar/read.c

```diff
@@ -182,7 +184,7 @@ read_archive(struct bsdtar *bsdtar, char mode, struct archive *writer)
 	if (reader_options != NULL) {
 		char *p;
 		/* Set default read options. */
-		p = malloc(sizeof(IGNORE_WRONG_MODULE_NAME)
+		p = (char*)malloc(sizeof(IGNORE_WRONG_MODULE_NAME)
```

Comment:
What error or warning did you get for this?  I did not think any compiler warned about casts to/from void *, but I know that VS is a little unusual.